From: Marcel Pennewiss <gentoo@pennewiss.de>
Date: Thu Apr 12 12:20:15 CEST 2012
Subject: fix as-needed

fails to compile with --as-needed LDFLAG caused by 
missing HomerGAPI dependency for libHomerMultimedia

--- HomerMultimedia/libHomerMultimedia/CMakeLists.txt
+++ HomerMultimedia/libHomerMultimedia/CMakeLists.txt
@@ -144,6 +144,7 @@
 IF (${BUILD} MATCHES "Default")
 	SET (LIBS_LINUX
 		HomerBase
+		HomerGAPI
 		HomerMonitor
 		${ALSA_LIBRARIES}
 		avdevice
